client/lib/media/actions.js

```diff
@@ -150,7 +151,8 @@ function uploadFiles( uploader, files, siteId ) {
 		Dispatcher.handleViewAction( {
 			type: 'CREATE_MEDIA_ITEM',
 			siteId: siteId,
-			data: transientMedia
+			data: transientMedia,
+			site: site
```

Comment:
you can use shorthand property names here and have only `site`.

Reply:
Nice catch @rodrigoi 👍 It was fixed, thank you!